CONTRIBUTING.md

```diff
@@ -1,12 +1,13 @@
 - [Introduction](#introduction)
 - [The contribution process](#the-contribution-process)
-  * [Submitting pull requests](#submitting-pull-requests)
-  * [Ensuring code quality](#ensuring-code-quality)
-    1. [Coding style](#coding-style)
-    1. [Code analysis and unit testing](#code-analysis-and-unit-testing)
+  * [Preparing pull requests](#preparing-pull-requests)
+    1. [Checking the coding style](#checking-the-coding-style)
+    1. [Unit testing](#unit-testing)
     1. [Building the documentation](#building-the-documentation)
+    1. [Signing your work](#signing-your-work)
     1. [Automatic code formatting](#automatic-code-formatting)
```

Comment:
I think we should do `black` first then signoff PR, otherwise you need to signoff again after `black`.
Thanks.

Reply:
thanks updated